dap_aria_mapping/pipeline/embeddings/make_embeddings.py

```diff
@@ -0,0 +1,98 @@
+from dap_aria_mapping import BUCKET_NAME, PROJECT_DIR, logger
+from toolz import pipe
+from itertools import chain
+from functools import partial
+import pickle, argparse
+import boto3
+from copy import deepcopy
+
+from dap_aria_mapping.utils.semantics import (
+    get_sample,
+    filter_entities,
+)
+from dap_aria_mapping.getters.openalex import get_openalex_entities
+from dap_aria_mapping.getters.patents import get_patent_entities
+from sentence_transformers import SentenceTransformer
+from torch import cuda
+
+MODEL_NAME = "sentence-transformers/allenai-specter"
+
+parser = argparse.ArgumentParser(
+    description="Create embeddings for OpenAlex entities",
+    formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+)
+parser.add_argument(
+    "--threshold", default=80, type=int, help="Threshold for OpenAlex scores"
+)
+parser.add_argument(
+    "--num_articles", default=3_000, type=int, help="Number of articles to sample"
```

Comment:
is this 3000 meant to have an _?

Reply:
It's convenience notation, preferably for large numbers (which this is not) but imo still helpful for small ints. See [this](https://peps.python.org/pep-0515/) for more.